### DISCUSSION TITLE
Qual é o padrão correto para máscara de telefone?

### DISCUSSION THREAD

**Discussion by drianoaz** [frontendbr/forum#1524] [Discussão]
Depois que houve um o acréscimo de um dígito em números de telefonia móvel, em todo site ou lugar que vejo, estão utilizam um formato diferente para a máscara de telefone. Exemplos:

- (99) 9999-9999
- (99) 9.9999-9999
- (99) 9-9999-9999
- (99) 9 9999-9999

Alguém sabe me dizer se há de fato um padrão para utilizarmos? Há alguma documentação da Anatel ou alguma coisa nesse sentido que fale sobre isso? <s>(fui incapaz de encontrar)</s> Se não há um padrão, qual vocês estão utilizando?

**Comment by victorferreira** [comment]
Até onde eu sei, após o código de área, a separação entre os número é arbitrária. Ou seja, não tem nenhum significado especial. O que eu costumo fazer é seguir o que aparece no meu telefone e marcar os códigos de país e região "+55 (11) ..." e separar apenas os últimos quatro dígitos.

```
+55 (99) 99999-9999
```

**Comment by lucasferreiralimax** [comment]
Também compartilho da mesma opinião do @victorferreira 

Assim já seria ótimo **+55 (99) 99999-9999**

**Comment by kivervinicius** [comment]
@drianoaz  eu vejo que o pessoal ta separando o primeiro 9 geralmente em impressos pois ainda  (principalmente onde foi recente a mudança) muitos se confundem, eu acho que muito é um tanto faz, rs, veja onde será mostrado o telefone, se for num sistema (input) acredito q o certo é como os colegas disseram, agora se for em midia eu ainda optaria por separar com um espaço o primeiro 9

**Comment by brunoak** [comment]
Cara semana passada um professor meu estava falando sobre isso, segundo ele o correto seria assim: (99) 999-999-999, pois multiplus de 3 é mais facil para decorar.

**Comment by drianoaz** [comment]
@brunoak eita, no mínimo curioso. Será que estamos todos errados esse tempo todo?

**Comment by kavalcante** [comment]
Eu acredito que a forma descrita acima: (99) 999-999-999 seja o futuro. Mas hoje eu utilizo assim: (99) 9 9999-9999.

**Comment by rafaelsevla** [comment]
eu utilizo normalmente (99) 9999-9999 com possibilidade de ficar (99) 99999-9999, eu uso esse regex aqui pra fazer: https://github.com/rafaelsevla/universal-utilities/blob/master/vanilla-js-masks/app.js#L21

**Comment by evertonthepaula** [comment]
Eu uso (99) 9999-99-999

Você tem que escrever como se fala, teste seus modelos mentais falando o número e percebendo o que é mais fácil de falar, logo também mais fácil de lembrar:

(99) 999-999-999
(99) 99999-9999
(99) 9-9999-9999
(99) 9999-99-999

Para facilitar use um telefone real.
